tests/unit/containers/tutor-home-page/subjects-step/SubjectsStep.spec.jsx

```diff
@@ -143,4 +143,82 @@ describe('SubjectsStep renders components', () => {
     expect(categoryField).toHaveValue('')
     expect(subjectField).toHaveValue('')
   })
+  it('should add subject when clicking the add button', async () => {
+    const categoryField = screen.getByLabelText(
+      'becomeTutor.categories.mainSubjectsLabel'
+    )
+    const subjectField = screen.getByLabelText(
+      'becomeTutor.categories.subjectLabel'
+    )
+    const addButton = screen.getByText('becomeTutor.categories.btnText')
+
+    userEvent.click(categoryField)
+    userEvent.click(screen.getByText('Category 1'))
+    userEvent.click(subjectField)
+    await waitFor(() => {
+      userEvent.click(screen.getByText('Subject 1'))
+    })
+
+    userEvent.click(addButton)
+
+    const chip = screen.queryByTestId('chip')
+    expect(chip).toHaveTextContent('Subject 1')
+  })
+
+  it('should show error message when the same subject is added', async () => {
+    const categoryField = screen.getByLabelText(
+      'becomeTutor.categories.mainSubjectsLabel'
+    )
+    const subjectField = screen.getByLabelText(
+      'becomeTutor.categories.subjectLabel'
+    )
+    const addButton = screen.getByText('becomeTutor.categories.btnText')
+
+    //first add
+    userEvent.click(categoryField)
+    userEvent.click(screen.getByText('Category 1'))
+    await waitFor(() => {
+      userEvent.click(subjectField)
+    })
+    await waitFor(() => {
+      userEvent.click(screen.getByText('Subject 1'))
+    })
+    userEvent.click(addButton)
+    const chip = screen.queryByTestId('chip')
+    expect(chip).toHaveTextContent('Subject 1')
+    //second add

```

Comment:
Maybe better to delete comments? ```it``` description explains everything well!

Reply:
I checked it and I guess that it can be clearer with these comments because we have too many events of click and won't confuse the situation we can have separate operations for adding subjects.